server/memphis_handlers_monitoring.go

```diff
@@ -443,9 +483,15 @@ cleanup:
 		})
 	}
 
-	sort.Slice(resMsgs, func(i, j int) bool {
-		return resMsgs[j].TimeSent.Before(resMsgs[i].TimeSent)
-	})
+	if getAll {
```

Comment:
it is looks like the same code in the if block and in the else block

Reply:
opposite order of logs - see 'i' and 'j' switch places